django/thunderstore/repository/migrations/0023_auto_20201220_1321.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.1.4 on 2020-12-20 13:21
+
+from django.db import migrations, models
+from django.db.models import Value
+from django.db.models.functions import Replace
+
+
+def set_default(apps, schema_editor):
+    for model_name in ("Package", "PackageVersion"):
+        model = apps.get_model("repository", model_name)
+        model.objects.update(display_name=Replace("name", Value("_"), Value(" ")))
```

Comment:
Even as nice it is for performance, using the update query is _usually_ a bad idea given how it does not run pre or post-save hooks.

I don't remember what hooks we have in use here exactly and if they are crucial, so I'd suggest you do one of:
- Do a per-object loop and call `save` on them rather than the update query
- Somehow batch-call the hooks when doinog the update query (no idea if this is supported by Django)
- Verify that there's no chance of missing a hook or logic that's run in save methods that has relevance. Something that usually is missed due to these is timestamps with `auto_now` will for example not get updated.

Reply:
Keep in mind that the `save` function might be overwritten in the models and contain logic as well which does not get called when `update` is called.